ldap_auth_provider.py

```diff
@@ -404,6 +404,74 @@ class _LdapConfig(object):
 
         return ldap_config
 
+    def _get_server(self, get_info: Optional[str] = None) -> ldap3.ServerPool:
+        """Constructs ServerPool from configured LDAP URIs
+
+        Args:
+            get_info (str, optional): specifies if the server schema and server
+            specific info must be read. Defaults to None.
+
+        Returns:
+            [ServerPool]: servers groupped to ServerPool
+        """
+        return ldap3.ServerPool(
+            [
+                ldap3.Server(
+                    uri,
+                    get_info=get_info,
+                    tls=ldap3.Tls(validate=ssl.CERT_REQUIRED)
```

Comment:
it seems like the `ldap3.Tls` constructor isn't trivial. Is there a good reason to use a separate Tls instance for each server?